Plots/MapProjections/NCL_proj_3.py

```diff
@@ -1,6 +1,7 @@
+
 """
-NCL_proj_3.py
-=============
+NCL_proj_3_lg.py
```

Comment:
I think my local version is from an older version of the code. I am currently going through and comparing things like this to make sure any additional edits are not detrimental (like in this one) and are formatted correctly. Most seem to be relatively quick and simple fixes, but I think I will remove this PR and reopen it with completely new pulls of the files from the repo just to completely ensure that there aren't any bigger differences that just haven't been caught yet, and I don't want to mess up any builds because of having a different version of a code locally for whatever reason. 

The new PR will most likely be published tomorrow AM, but I will work on getting everything re-done in preparation for that tonight.

Reply:
That was a slight lie, I'm not going to delete this PR. I have cleared out the versions of the codes that I have locally and pulled them from the master. I'm going to now re upload them to this PR, make sure the only differences are from the cbar edits and then go ahead with adding in the cbar changes again. You will probably get a couple of emails during this process, so sorry for the spam in advance!